### PR TITLE
https://blueprints.launchpad.net/or/+spec/distributed-power: Distributed power cab display and controls

### PR DESCRIPTION
**Pull request by Csantucci**
See http://www.elvastower.com/forums/index.php?/topic/26249-remote-control-dpu-units/page__view__findpost__p__279652

### CODE REVIEW

File: Source/Documentation/Manual/cabs.rst [hunk 1]
```diff
@@ -403,6 +403,59 @@ Here is an example of use of the odometer control blocks within a .cvf file::
...
+         FullTable True
+		)
+      Units ( KM_PER_HOUR )
+		ORTSDisplay ( 1 )
+		ORTSScreenPage ( "2300-0" )
+   )  
+
+Here below an example of the output of the above control.
+
+.. image:: images/cabs-distributed-power.png
```
Comment (Sharpe49) [inline Source/Documentation/Manual/cabs.rst:429]: Image seems to be missing from the commit

File: Source/Documentation/Manual/cabs.rst [hunk 2]
```diff
@@ -403,6 +403,59 @@ Here is an example of use of the odometer control blocks within a .cvf file::
...
+
+When parameter FullTable is set to False, only the first 6 lines 
+are displayed.
+
+For every keyboard command related to Distributed Power, also a cabview control 
+is available. Here a list of the self-explaining controls::
```
Comment (Sharpe49) [inline Source/Documentation/Manual/cabs.rst:435]:
```suggestion
For every keyboard command related to Distributed Power, a cabview control 
is also available. Here's a list of the cabview controls::
```

File: Source/Documentation/Manual/cabs.rst (hunk 1 above)
Comment (Csantucci) [reply Source/Documentation/Manual/cabs.rst:429]: Missing picture added, thanks

**Review by cjakeman — APPROVED** [review]
Hi Carlo,

All looks good.

Approved with thanks,

Chris

**Comment by Csantucci** [comment]
> Hi Carlo,
> 
> All looks good.
> 
> Approved with thanks,
> 
> Chris

Thanks Chris,
very kind of you.

Carlo

**Pull request merged**